docs/pages/guides/deliver-input.md

```diff
@@ -0,0 +1,80 @@
+# How to deliver input streams
+
+Live Compositor currently supports 2 input types:
+- RTP
+- MP4 (not supported in Membrane Framework plugin)
+
+MP4 support is useful if you want to add some prerecorded assets, but for most streaming use cases RTP protocol will be a primary choice.
+
+Our RTP implementation supports following codecs:
+- H264 for video
+- AAC and Opus for audio (AAC is not supported via Membrane Framework plugin)
+
+### RTP over TCP vs UDP
+
+RTP streams can be delivered over TCP or UDP. Depending on your use case different choice might make more sense, but in general we recommend using TCP if possible.
+
+What to choose?
+- If you are using MembraneFramework plugin all communication already happens over TCP. Currently, we do not support any way to configure it.
+- Some of the popular multimedia tools do not support RTP over TCP e.g. FFmpeg.
+- UDP should only be used for communication on localhost. We do not support retransmission or packet reordering, so if you use it in unreliable network it might lead to unexpected behavior.
```

Comment:
```suggestion
- UDP should only be used for communication on localhost. We do not support retransmission or packet reordering, so if you use it in an unreliable network it might lead to unexpected behavior.
```